### PR TITLE
week-2 part 3: encode return data in JSON

### PR DESCRIPTION
**Pull request by AzfaarQureshi**
Changing `doGet` in the dataServlet to return JSON string

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -13,20 +13,47 @@
...
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+    messages.add(
+        "Azfaar Qureshi");
```
Comment (supersam654) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Nit: please make lines 32 and 33 into a singe line.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ function addRandomGreeting() {
 }
 
 function fetchName() {
-  fetch('/data').then(response => response.text()).then((name) => {
-    document.getElementById('name-container').innerHTML = name;
-  });
+  const messageContainer = document.getElementById('comments')
+  fetch('/data').then(response => response.json()).then(name => name.reduce((acc, curVal) =>
+    acc + "<p>" + curVal + "</p></br>")
+  ).then( (comments) => {
```
Comment (supersam654) [inline portfolio/src/main/webapp/script.js]: Nit: `( (comments) =>` -> `(comments =>` (no space or extra parens).

**Review by supersam654 — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (AzfaarQureshi) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: done!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (AzfaarQureshi) [reply portfolio/src/main/webapp/script.js]: done!

**Pull request merged**